cardano-cli/test/cardano-cli-golden/files/golden/help/governance_action_create-constitution-update.cli

```diff
@@ -0,0 +1,7 @@
+Usage: cardano-cli governance action create-constitution-update --out-file FILE
```

Comment:
What is the content of the out-file this way??

I think it should be at minimum something like:

```
cardano-cli governance action create-constitution-update
--proposal-hash STRING
--out-file
```
where we can specify what algorithm to use or ideally provide a tool on the cli. 

I know it mostly depends on ledger but I have already discussed with Alexey and Jared. User should provide a URL to the document proposed and its hash, and ideally a compressed version of the proposal as auxiliary data. this way the compressed version will always be ON-CHAIN 

```
cardano-cli governance action create-constitution-update
--proposal-url  URL
--proposal-hash STRING
--proposal-compressed-file FILE? RAW-BYTES?
--out-file
```